app/pages/sections/catalogue-solutions/order-item/controller.js

```diff
@@ -0,0 +1,15 @@
+import { getContext } from './contextCreator';
+import { getSolution } from '../select/recipient/controller';
+
+export const getOrderItemContext = async ({
+  selectedSolutionId,
+  selectedRecipientId,
+  recipients,
+  accessToken,
+}) => {
+  const solutionName = await getSolution({ solutionId: selectedSolutionId, accessToken }).name;
+  const serviceRecipientName = recipients.find(
```

Comment:
I thought we were going to be calling an endpoint to find the recipient rather than do it ourselves.